sdks/python/apache_beam/io/avroio.py

```diff
@@ -129,6 +127,12 @@ def read_block_from_file(f, codec, schema, expected_sync_marker):
 
     Args:
       f: Avro file to read.
+      codec: Decompression codec to use.
+        Supported codecs: 'null', 'deflate', 'snappy'
+      schema: Avro Schema definition represented as JSON string.
+      expected_sync_marker: Avro synchronization marker.
```

Comment:
Would this fit in two lines?

```
      expected_sync_marker: Avro synchronization marker. If the block's sync
        marker does not match with this parameter then ValueError is thrown.
```

Second question (to @sb2nov), should we start annotating code blocks inside comments for the them to show up nicely in pydocs? (like ValueError in this case.)